Packages/PackageSources/meridian-simcache/html_ui/InGamePanels/SimCache/SimCache.html

```diff
@@ -22,14 +20,31 @@
 
 <body id="SimCache" class="border-box">
     <ingamepanel-simcache>
-        <ingame-ui id="SimCache_Frame" panel-id="PANEL_SIMCACHE" class="ingameUiFrame panelInvisible" title="SimCache" content-fit="true" min-width="40" resize="x">
-            <ui-element style="display: flex">
-                <div id="simcache-placeholder">
-                    <h1>SimCache Title Placeholder</h1>
-                    <p>This paragraph is placeholder.</p>
+        <ingame-ui id="SimCache_Frame" panel-id="PANEL_SIMCACHE" class="ingameUiFrame panelInvisible" title="SimCache"
+            content-fit="true" min-width="40" resize="x">
```

Comment:
Is this intended to be on a new line?

Reply:
I used Visual Studio Code's default HTML formatter -- it thought the line was too long and it got moved.  I can move it back so it's a single line?

Reply:
Ah okay! The auto formatter makes the review a little harder to parse with all the random spacing/line break changes alongside the actual logic 😛 . Keep it this way for this PR, but maybe see if there's a way to avoid this so the diffs are cleaner?